UPGRADE-6.3.md

```diff
@@ -66,3 +66,8 @@ Validator
 ---------
 
  * Implementing the `ConstraintViolationInterface` without implementing the `getConstraint()` method is deprecated
+
+Serializer
+----------
+
+ * Deprecate the `Symfony\Component\Serializer\Exception\MissingConstructorArgumentsException` class, use `Symfony\Component\Serializer\Exception\MissingConstructorArgumentException` instead
```

Comment:
let's be short:
`Deprecate MissingConstructorArgumentsException in favor of MissingConstructorArgumentException`

Reply:
Done